### PR TITLE
Fixed #1699: Permission dialog error fixed in BeneficiaryAddOptionsFragment.kt

### LINKED ISSUE #1699: Permission Dialog shows 2 times in Upload QR
**Summary:**
When user goes to add beneficiary and selects Upload QR option then if app does not have storage permissions it makes 2 repetitive dialogs, instead of just one.

https://user-images.githubusercontent.com/70195106/104501455-bff10100-5605-11eb-84b4-5044bee2d777.mp4

**Steps to reproduce:**
1 Open Beneficiary
2 Add Beneficiary
3 Select Upload QR option

**Expected behaviour:**
Only one permission dialog show be visible

**Device and Android version:**
Any

### PR DESCRIPTION
**Pull request by PatelVatsalB21**
Fixes #1699 
Permission dialog has been simplified and also snackbar is visible now.

https://user-images.githubusercontent.com/70195106/104503089-9d131c80-5606-11eb-9eb2-61d00da3d9e2.mp4

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/mifos/mobile/ui/fragments/BeneficiaryAddOptionsFragment.kt [hunk 1]
```diff
@@ -119,21 +115,9 @@ class BeneficiaryAddOptionsFragment : BaseFragment() {
...
-                        Constants.PERMISSIONS_STORAGE_STATUS)
-            }
-            RequestAccessType.EXTERNAL_STORAGE_WRITE -> {
-                requestPermission((activity as BaseActivity?)!!,
-                        Manifest.permission.WRITE_EXTERNAL_STORAGE,
-                        Constants.PERMISSIONS_REQUEST_WRITE_EXTERNAL_STORAGE,
-                        resources.getString(R.string.dialog_message_storage_permission_denied_prompt),
-                        resources.getString(R.string.dialog_message_write_storage_permission_never_ask_again),
-                        Constants.PERMISSIONS_STORAGE_STATUS)
+                requestPermissions( arrayOf(Manifest.permission.READ_EXTERNAL_STORAGE,
```
Comment (divyank00) [inline app/src/main/java/org/mifos/mobile/ui/fragments/BeneficiaryAddOptionsFragment.kt:118]: Why did you remove the first parameter: activity?

**Review by divyank00 — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/mifos/mobile/ui/fragments/BeneficiaryAddOptionsFragment.kt (hunk 1 above)
Comment (PatelVatsalB21) [reply app/src/main/java/org/mifos/mobile/ui/fragments/BeneficiaryAddOptionsFragment.kt:118]: That is for custom text in permission dialog also it is written 2 times so calls dialog 2 times so I added code for multiple permission with default text.
